testing/scripts/Makefile

```diff
@@ -18,14 +18,10 @@ kind_build_operator:
 
 kind_build_images: kind_build_engine kind_build_operator
 
-install_helm:
```

Comment:
Installation tests fail as `install_helm` command is required, so either script needs to be modified or install_helm should be added